static/app/gettingStartedDocs/android/android.tsx

```diff
@@ -28,10 +29,14 @@ export const steps = ({
     configurations: [
       {
         language: 'groovy',
+        loading: sourcePackageRegistries?.isLoading,
         code: `
 plugins {
   id "com.android.application" // should be in the same module
-  id "io.sentry.android.gradle" version "3.12.0"
+  id "io.sentry.android.gradle" version "${
+    sourcePackageRegistries?.data?.['sentry.java.android.gradle-plugin']?.version ??
```

Comment:
Instead of passing in `sourcePackageRegistries` entirely could we just pass in `isLoading` and `version`? Then all that a steps component is responsible for is a default version and the steps component doesn't need to understand how `sourcePackageRegistries` is shaped.

```
export const steps = ({
  dsn,
  isVersionLoading,
  version,
}: Partial<
```

Reply:
At first, I believed we could achieve this, and it was implemented accordingly. However, my perspective shifted when I noticed that in JAVA, for instance, we showcase versions of distinct packages.